Tasks/DownloadBuildArtifacts/main.ts

```diff
@@ -231,6 +231,16 @@ async function main(): Promise<void> {
                         reject(reason);
                     }));
                 }
+                else if (artifact.resource.type.toLowerCase() === "drop") {
+                    let properties = {
+                        artifacttype: artifact.resource.type,
+                        artifactname: artifact.name,
+                        itempattern: itemPattern,
+                        manifestid: artifact.resource.data
+                    };
+
+                    tl.command("artifact.download", properties, downloadPath);
```

Comment:
Does `console.log(tl.loc('ArtifactsSuccessfullyDownloaded', downloadPath));` below the diff execute before the artifact is downloaded? Not sure if there's a way to monitor completion/success.

Reply:
that's why artifact.download needs to be removed